lib/lightstep/tracer/span.rb

```diff
@@ -0,0 +1,132 @@
+# TODO(ngauthier@gmail.com) Separate Span and SpanContext under a getter according
+# to the spec. Baggage moves to span context.
+module LightStep
+  # Span represents an OpenTracer Span
+  #
+  # See http://www.opentracing.io for more information.
+  class Span
+    # UnsupportedValueTypeError is raised when a tag value is not a supported type
+    class UnsupportedValueTypeError < LightStep::Error; end
+
+    # The guid of the span
+    attr_reader :guid
+    # Tags on the span
+    attr_reader :tags
```

Comment:
`attr_reader` is a getter. `attr_accessor` is a getter and setter. `attr_writer` is a setter. So I've only publicized the getters here.